src/components/common/Extensions/UserNavigation/partials/UserSubmenu/blocks/MenuList/MenuList.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { PropsWithChildren } from 'react';
```

Comment:
What's the idea with blocks? How are they different from partials?

Reply:
My thinking was that 'blocks' are components which are reusable among the `partials`, but come to think of it, we can just put them into the partials directory.